extension/package.json

```diff
@@ -404,6 +412,12 @@
                     "default": false,
                     "description": "If enabled log statements will be printed to console.",
                     "scope": "resource"
+                },
+                "NAB.XliffTSVExportPath": {
```

Comment:
This looks a bit funny in settings UI...

but spaces seems to be fine in settings name, so maybe we should change this to something like
```suggestion
                "NAB.Xliff tsv ExportPath": {
```